eng/common/tools.ps1

```diff
@@ -184,7 +183,7 @@ function InstallDotNetSdk([string] $dotnetRoot, [string] $version, [string] $arc
   InstallDotNet $dotnetRoot $version $architecture
 }
 
-function InstallDotNet([string] $dotnetRoot, [string] $version, [string] $architecture = "", [string] $runtime = "", [bool] $skipNonVersionedFiles = $false) {
+function InstallDotNet([string] $dotnetRoot, [string] $version, [string] $architecture = "", [string] $runtime = "", [bool] $skipNonVersionedFiles = $true) {
```

Comment:
This is to workaround the issue with installing the unstable version 3.1.102 SDK and calling `build.cmd` multiple times. We should be able to remove this once 3.1.102 ships \ dotnet-install can install unstable zips that contain stable SDKs or runtime as discussed in the email thread.

Reply:
Any change made under eng/common/ will be obliterated as part of the next Arcade dependency update. We should _never_ be editing these files except in short-term emergencies i.e. when we can't wait for an Arcade fix.

Reply:
> We should never be editing these files except in short-term

Yup. Like I said, we should be able to undo this once 3.1.102 ships, which is fairly soon, no?

Reply:
February 18